src/julia_gc.c

```diff
@@ -817,6 +817,26 @@ void InitBags(UInt initial_size, Bag * stack_bottom, UInt stack_align)
     max_pool_obj_size = jl_gc_max_internal_obj_size();
     jl_gc_enable_conservative_gc_support();
     jl_init();
+
+    // Import GAPTypes module to have access
+    // to GapObj abstract type.
```

Comment:
Weird wrapping, perhaps that comment could go into a single line? (Clearly, this is *very* important ;-)